src/components/common/TokenInputField/TokenInputField.tsx

```diff
@@ -33,11 +34,12 @@ export const TokenInputField: FC<Props> = ({
   isError,
   className,
   onKeystroke,
-  placeholder = 'Enter Amount',
+  placeholder = i18n.t('common.placeholders.placeholder2'),
```

Comment:
so why use the hook at all?

Reply:
As we talked, we are now using t function from hook only and pass it down to functions if needed.